src/main/java/org/gridsuite/cases/importer/job/CassandraConnector.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.cases.importer.job;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.Metadata;
+import com.datastax.driver.core.Session;
+
+/**
+ * @author Nicolas Noir <nicolas.noir at rte-france.com>
+ */
+public class CassandraConnector
+{
+    private Cluster cluster;
```

Comment:
Indentation is not correct. Checkstyle should have warned you for this. Did you disable checkstyle?

Reply:
Yes, checkstyle was not enabled, the pom.xml did not contain all plugins. I'll fix this.

Reply:
Inheritance from powsybl parent pom added